app/assets/javascripts/dashboard/spotlight_view.js

```diff
@@ -117,36 +115,26 @@ type Props = {
 
 type State = {
   datasets: Array<APIMaybeUnimportedDatasetType>,
+  hasOrganisations: boolean,
   isLoading: boolean,
 };
 
 class SpotlightView extends React.PureComponent<Props, State> {
   state = {
     datasets: [],
+    hasOrganisations: true,
     isLoading: true,
   };
 
   componentDidMount() {
     this.fetchData();
-    this.maybeRedirectToOnboarding();
-  }
-
-  async maybeRedirectToOnboarding() {
-    const organizations = await getOrganizations();
-    if (organizations.length > 0) {
-      return;
-    }
-
-    message.info(messages["setup.redirect"]);
```

Comment:
Please remove the key-value pair for `setup.redirect` from `messages.js` since this is not needed anymore.